### ISSUE TITLE
About the training data

### ISSUE THREAD

**Issue by mycrazycracy** [phonexiaresearch/VBx-training-recipe#4]
Hi,

Thanks for the great code. I have one question when I read through the code. 

In train.sh, it claims that the total number of speakers in the training set is 8199. I'm not sure how this number comes. In run.sh, the training data contains voxceleb 1, voxceleb 2 dev and voxceleb-cn. VoxCeleb1 contains 1211/40 speakers in the dev/test set. VoxCeleb2 contains 5994/118 speakers in dev/test set. But what is voxceleb-cn? Is it cn-celeb which contains 800/200 speakers in the dev/test set?

Any advice would be appreciated! Thank you!

**Comment by Jamiroquai88** [comment]
Hi,

- voxceleb1 - at [this](https://github.com/phonexiaresearch/VBx-training-recipe/blob/81a5ade8421d6c5d1bf82a0b24faa664b9ad3c86/run.sh#L38) line you can see that we use a list as a parameter to remove voxceleb1 test speakers (voxceleb1 doesn't have dev/eval split)
- voxceleb-cn is this one https://deepai.org/publication/cn-celeb-a-challenging-chinese-speaker-recognition-dataset

**Comment by Jamiroquai88** [comment]
@MichalKlco

**Comment by mycrazycracy** [comment]
Thank you for the useful information! We often call voxceleb-cn as "CN-Celeb" :-P
